Bottom: e47ce6bd33efca8af994b659c2c4f791119bdcc0
Top:    2c67bfb9a836113fbc4d4d759eafbe4d9c3701db
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-25 11:31:35 +0800

vhost: batch descriptor getting

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index b91bf6e..72e5638 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -128,7 +128,7 @@ struct vhost_virtqueue {
 	bool log_used;
 	u64 log_addr;
 
-	struct iovec iov[UIO_MAXIOV];
+	struct iovec iov[2 * UIO_MAXIOV];
 	struct iovec iotlb_iov[64];
 	struct iovec *indirect;
 	struct vring_used_elem *heads;
